Gemfile

```diff
@@ -38,6 +38,8 @@ gem 'deacon', '~> 1.0'
 gem 'webpack-rails', '~> 0.9.8'
 gem 'mail', '~> 2.6'
 gem 'sshkey', '~> 1.9'
+gem 'bootstrap-datepicker-rails'
```

Comment:
I know, we don't like gemified assets anymore. But I couldn't get the i18n working with the node version. And this is already packaged for plugins.